pkg/apiserver/certificate/cacert_controller.go

```diff
@@ -173,11 +191,14 @@ func (c *CACertController) syncConversionWebhooks(caCert []byte) error {
 			crdDef.Spec.Conversion.Webhook.ClientConfig.CABundle = caCert
 		}
 		if updated {
-			if _, err := c.apiExtensionClient.ApiextensionsV1().CustomResourceDefinitions().Update(context.TODO(), crdDef, metav1.UpdateOptions{}); err != nil {
+			if _, err := c.apiExtensionClient.ApiextensionsV1().CustomResourceDefinitions().Update(context.TODO(), &crdDef, metav1.UpdateOptions{}); err != nil {
 				return fmt.Errorf("error updating antrea CA cert of CustomResourceDefinition %s: %v", name, err)
 			}
 		}
 	}
+	if unSupportedResources.Len() > 0 {
+		klog.InfoS("Unsupported CRD ConversionWebhooks are found", "CRDs", unSupportedResources.List())
```

Comment:
```suggestion
		klog.InfoS("Unsupported CRD ConversionWebhooks are found", "webhooks", unSupportedResources.List())
```
keep the format consistent, you use 'webhooks' in other logs.